tests/FormBuilderTest.php

```diff
@@ -172,6 +172,27 @@ public function testRenderCheckboxWithOldInput()
 		$this->assertEquals($expected, $result);
 	}
 
+	public function testRenderCheckboxArrayWithOldInput()
+	{
+		$oldInput = Mockery::mock('AdamWathan\Form\OldInput\OldInputInterface');
+		$oldInput->shouldReceive('hasOldInput')->andReturn(true);
+		$oldInput->shouldReceive('getOldInput')->with('favourite_foods')->andReturn(array(0 => 'fish', 1 => 'chips'));
```

Comment:
```['thank' => 'goodness']```

I only used `array()` because you use it everywhere else in package lol.

I'll remove the keys though.